README.md

```diff
@@ -28,7 +28,8 @@ import "github.com/gin-contrib/sessions"
 
 #### cookie-based
 
-[embedmd]:# (example_cookie/main.go go)
+[embedmd]: # "example_cookie/main.go go"
+
```

Comment:
Are these necessary diffs?  I think `[embedmd]: # (...)` is a special format for [embedmd](https://github.com/campoy/embedmd).

Reply:
Ah, you did this commit for travis!  See [the log](https://travis-ci.org/gin-contrib/sessions/jobs/375171855), it says

<img width="163" alt="2018-05-09 8 57 53" src="https://user-images.githubusercontent.com/1239245/39788808-247e0dec-5367-11e8-9a43-251684a383d2.png">

This reports it discovered a redundant empty line here in README.  Fix it.

Reply:
fixed